### PR TITLE
Shivansh/docker script fix

### PR DESCRIPTION
**Pull request by ShivanshGahlot**

### CODE REVIEW

File: docker/yb-voyager-docker [hunk 1]
```diff
@@ -27,16 +29,46 @@ do
...
+			fi
+			argv[$j]='/data-dir'
+		fi
+	fi	
+
     i=$(( $i + 1))
 done
 
-dockerCmd="docker run ${vol} -it --network=host --rm --privileged ${platform} yugabytedb/yb-voyager yb-voyager ${argv[*]}"
+dockerCmd="docker run ${exported_vars} ${export_vol} ${data_vol} ${aws_vol} -it --network=host --rm --privileged ${platform} yugabytedb/yb-voyager yb-voyager ${argv[*]}"
```
Comment (amit-yb) [inline docker/yb-voyager-docker:71]: Undefined var `aws_vol`.

File: docker/yb-voyager-docker
Comment (amit-yb) [inline docker/yb-voyager-docker:1]: This file uses both 8 spaces and tabs for indentation which leads to inconsistent code layout. Let's standardise on tabs. Please fix the entire file by replacing all occurrences of 8 spaces with tabs.

File: docker/yb-voyager-docker (hunk 1 above)
Comment (ShivanshGahlot) [reply docker/yb-voyager-docker]: Done

File: docker/yb-voyager-docker [hunk 2]
```diff
@@ -27,16 +29,46 @@ do
...
+		else 
+			# If the data-dir is not an S3 bucket
+			# If the OS is macOS
+			if [[ $OSTYPE == 'darwin'* ]]; then
+
+				# Delete the old volume and create a new one it its place attached to the user's data-dir
+				docker volume rm voyager-data-dir
+				docker volume create --driver local --opt type=none --opt device="${data_dir}" --opt o=bind voyager-data-dir 
+		
+				data_vol="${vol} -v voyager-data-dir:/data-dir"
```
[the remark this answers is not in the document]
Comment (ShivanshGahlot) [reply docker/yb-voyager-docker]: Done

File: docker/yb-voyager-docker
Comment (ShivanshGahlot) [reply docker/yb-voyager-docker:1]: I specifically use tabs always. Dont know how this is happening🤔

**Pull request merged**